pytype/tools/merge_pyi/merge_pyi.py

```diff
@@ -626,6 +626,12 @@ def insert_comment_annotation(self, src_sig, pyi_sig):
       return []
 
   def can_annotate(self, src_sig):
+    if src_sig.short_name.startswith('__') and \
+            src_sig.short_name.endswith('__') and \
+            src_sig.short_name != "__init__":
+      self.logger.warning('builtin function, skipping %s', src_sig)
```

Comment:
nit: I find "builtin function" slightly confusing, since it makes me of think of builtins (https://docs.python.org/3/library/builtins.html). "Magic method" is a more common way of referring to these.

Reply:
Yeah, done.